src/app/submission/sections/container/section-container.component.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../../../styles/variables';
+
+:host /deep/ .card {
+  margin-bottom: $border-radius * 2;
```

Comment:
fixed

Reply:
It isn't fixed, you moved it to its own variable and [that is still based on the border radius](https://github.com/DSpace/dspace-angular/pull/279/files#diff-ade74f2136fdbe405029e30cefaec31cR29). Like I said:

>if someone sets the border-radius to 0, because they want hard corners, would you want the margin here to disappear?

Please base it on something related to margins or spacing like [`$spacer`](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L129), [`$paragraph-margin-bottom`](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L182), [`$card-spacer-y`](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L815), etc. any of those would be more suitable than `$border-radius`